databuilder/setup.py

```diff
@@ -1,25 +1,18 @@
 # Copyright Contributors to the Amundsen project.
 # SPDX-License-Identifier: Apache-2.0
-
+import os
 
 from setuptools import find_packages, setup
 
 __version__ = '4.4.1'
 
+requirements_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'requirements.txt')
+with open(requirements_path) as requirements_file:
+    requirements = requirements_file.readlines()
 
-requirements = [
```

Comment:
I thought for databuilder, we didn't want to specify exact deps (since it may be installed alongside other stuff) -- I think we should either update the requirements.txt to be less-specific, or remove this. WDYT?

Reply:
+1 on dorian's comment

Reply:
it doesn't really change the approach we had before but if you think that with this PR we want to stop pinning deps in databuilder (at least those generic ones) then I will remove version pinning.

Reply:
I have started to work on requirements.txt in databuilder per your suggestion. I didn't find a lot of deps there that are directly mentioned in any of the `extras_require` section. There were few I've moved (to atlas and rds) and my approach for the remaining ones would be not to remove pinning deps completely but instead replace `==` with `>=` . This way we at least we don't fall into regression issues if some library requires `lower` version than we've previously supported. Let me know your thoughts on that. @feng-tao @dorianj